.gitignore

```diff
@@ -15,6 +15,8 @@ tests/go-e2e-fileops/go-e2e-fileops
 tests/go-e2e-outgoing/go-e2e-outgoing
 tests/go-e2e-dns/go-e2e-dns
 
+mirrord-layer/tests/apps/app_go/19
```

Comment:
This is oddly specific

Reply:
Haha why? It's the binary of the go test app, like all the lines above it.

Reply:
Ah my mistake